.github/workflows/ci.yml

```diff
@@ -1362,6 +1362,42 @@ jobs:
           UV_TEST_PUBLISH_CLOUDSMITH_TOKEN: ${{ secrets.UV_TEST_PUBLISH_CLOUDSMITH_TOKEN }}
           UV_TEST_PUBLISH_PYTHON_VERSION: ${{ env.PYTHON_VERSION }}
 
+  integration-uv-build-backend:
+    timeout-minutes: 10
+    needs: build-binary-linux-libc
+    name: "integration test | uv_build"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Why do we need the full git history here?

Reply:
Thanks, TIL `fetch-depth: 0` fetches everything instead of setting the lowest possible fetch amount.

Reply:
Oh lol, yeah.